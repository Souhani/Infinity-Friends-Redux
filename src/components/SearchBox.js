const SearchBox = ({ onSearchFieldChange }) => {
  return (
    <div className="mb3">
      <input
        aria-label="search robots"
        type="search"
        placeholder="search friends"
        className="b--dark-blue bg-light-yellow pa2 input-reset ba  hover-bg-light-green hover-black"
        onChange={onSearchFieldChange}
      />
    </div>
  );
};

export default SearchBox;
