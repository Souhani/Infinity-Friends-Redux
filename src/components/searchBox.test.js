import React from 'react';
import { shallow } from 'enzyme';
 
import SearchBox from './SearchBox';

it ('testing SearchBox', () => {
    expect(shallow(<SearchBox />)).toMatchSnapshot();
})