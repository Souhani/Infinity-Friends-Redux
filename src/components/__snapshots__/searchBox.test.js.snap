// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing SearchBox 1`] = `
<div
  className="mb3"
>
  <input
    aria-label="search robots"
    className="b--dark-blue bg-light-yellow pa2 input-reset ba  hover-bg-light-green hover-black"
    placeholder="search friends"
    type="search"
  />
</div>
`;
